packages/graph/hash_graph/lib/graph/src/store/postgres/knowledge/entity/read.rs

```diff
@@ -21,7 +28,52 @@ impl<C: AsClient> crud::Read<PersistedEntity> for PostgresStore<C> {
         &self,
         filter: &'f Self::Query<'q>,
     ) -> Result<Vec<PersistedEntity>, QueryError> {
-        let mut compiler = SelectCompiler::with_default_selection();
+        let mut compiler = SelectCompiler::new();
+
+        compiler.add_selection_path(
+            &EntityQueryPath::Properties(None),
+            Distinctness::Indistinct,
+            None,
+        );
+        compiler.add_selection_path(
+            &EntityQueryPath::Id,
+            Distinctness::Distinct,
+            Some(Ordering::Ascending),
+        );
+        compiler.add_selection_path(
+            &EntityQueryPath::Version,
+            Distinctness::Distinct,
+            Some(Ordering::Descending),
+        );
+        compiler.add_selection_path(
+            &EntityQueryPath::Type(EntityTypeQueryPath::VersionedUri),
+            Distinctness::Indistinct,
+            None,
+        );
+        compiler.add_selection_path(&EntityQueryPath::OwnedById, Distinctness::Indistinct, None);
+        compiler.add_selection_path(
+            &EntityQueryPath::CreatedById,
+            Distinctness::Indistinct,
+            None,
+        );
+        compiler.add_selection_path(
+            &EntityQueryPath::UpdatedById,
+            Distinctness::Indistinct,
+            None,
+        );
+        compiler.add_selection_path(
+            &EntityQueryPath::LeftEntityId,
+            Distinctness::Indistinct,
+            None,
+        );
+        compiler.add_selection_path(
+            &EntityQueryPath::RightEntityId,
+            Distinctness::Indistinct,
+            None,
+        );
+        compiler.add_selection_path(&EntityQueryPath::LeftOrder, Distinctness::Indistinct, None);
```

Comment:
Isn't this literally the one we do want to order? 🤔

Reply:
I think we want to order by:
* Source Entity ID (left entity ID)
* Left Order

Reply:
But again maybe we don't even want to do that until we expose control of ordering through queries